### PR TITLE
LTD-5685-dbt-API-data-pull-config-changes

### PR DESCRIPTION
**Pull request by emileswarts**
### Aim

Add additional configurations to allow this codebase to run as a scheduled task on DBT Platform.

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 web: SWIG_LIB=/home/vcap/deps/0/apt/usr/share/swig4.0 CFLAGS=-I/home/vcap/deps/1/python/include/python3.9.18m pip3 install endesive==1.5.9 && python manage.py migrate && gunicorn --worker-class gevent -c api/conf/gconfig.py -b 0.0.0.0:$PORT api.conf.wsgi
 web-dbt-platform: python manage.py migrate && gunicorn -c api/conf/gconfig-dbt-platform.py -b 0.0.0.0:$PORT api.conf.wsgi
 dump-and-anonymise: python manage.py dump_and_anonymise
+rebuild-search-index: python manage.py search_index --rebuild -f
```
Comment (depsiatwal) [inline Procfile:4]: we don't want to run these in PROD 
is this the commands only ?

Comment (emileswarts) [reply Procfile:4]: Yeah this is a good point. It's currently not possible to run this in prod (job hasn't been deployed there) but we can restrict at this level as well. Will update

Comment (emileswarts) [reply Procfile:4]: Fix has been added

**Review by depsiatwal — APPROVED** [review]
lgtm

File: Procfile [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+seed-internal-users: ./bin/seed_internal_users.sh
```
Comment (saruniitr) [inline Procfile:5]: do we want to run this on `prod`?

Comment (depsiatwal) [reply Procfile:5]: it won't run on prod 

the command is there but if anyone attempts to run it exits 
https://github.com/uktrade/lite-api/blob/dev-migration-post-pause/bin/seed_internal_users.sh

Comment (saruniitr) [reply Procfile:5]: ok, may be best to remove this altogether.
We can invite internal users from UI.

Comment (depsiatwal) [reply Procfile:5]: St the moment it's part of the seed environment in non dev environments 
effectively the same  infrastructure we use currently

File: bin/seed_internal_users.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+if [[ "${ENV}" == "prod" ]]; then
+  echo "You can't re-index the prod OpenSearch instance"
```
Comment (saruniitr) [inline bin/seed_internal_users.sh:6]: this doesn't match with this command

**Review by saruniitr — APPROVED** [review]
error message need updating

File: bin/seed_internal_users.sh (hunk 3 above)
Comment (depsiatwal) [reply bin/seed_internal_users.sh]: well spotted changed

**Pull request merged**